src/plugins/new-ui/public/webcomponents/depth.html

```diff
@@ -16,9 +16,9 @@
         margin-top: -0.6vw;
       }
       #depth {
-        font-size: 1.8vw;
+        font-size: 1.35vw;
         font-weight: bold;
-        text-align: center;
+        text-align: left;
```

Comment:
While I understand trying to separate the units from the depth because the units are clickable, It is not very pleasing to the eye for the depth to be off center. Given that these clickable items in the display are not otherwise called out, I think it is okay to leave the units spaced right after the depth for now. We might instead move the imperial vs metric as a setting under UI.  We can leave the clickable unit as a sort of easter-egg for those that know it is there.